target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -348,11 +348,15 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"1:lan" "2:lan" "3:lan" "4:lan" "0:wan" "9@eth0"
 		;;
-	ubnt,acb-isp)
+	ubnt,aircube-isp)
 		ucidef_set_interface_wan "eth1"
 		ucidef_add_switch "switch0" \
 			"0@eth0" "2:lan:1" "3:lan:3" "4:lan:2"
 		;;
+	ubnt,aircube-ac)
```

Comment:
ac comes before isp ...

Reply:
thanks,
as we keep changes for acb-isp -> aircube-isp change separately - order would be correct for now.
ubnt,acb-isp comes first and then ubnt,aircube-ac


i will swap them around making aircube-ac first with mr that renames acb-isp -> aircube-isp.